TTS/vocoder/utils/generic_utils.py

```diff
@@ -61,40 +61,37 @@ def plot_results(y_hat, y, ap, global_step, name_prefix):
     return figures
 
 
-def to_camel(text):
+def to_camel(text, cap=True):
```

Comment:
Not sure what ```cap=True``` does?

Reply:
yeah - this was the first idea of avoiding capitalization of `WaveRNN`, but then I avoided passing it through to `to_camel` altogether. 

I've also modified the test config as @SanjaESC suggested to see if it now passes the checks.